builtin/gen/bindata.go

```diff
@@ -794,11 +794,13 @@ var _bindata = map[string]func() (*asset, error){
 // directory embedded in the file by go-bindata.
 // For example if you run go-bindata on data/... and data contains the
 // following hierarchy:
-//     data/
-//       foo.txt
-//       img/
-//         a.png
-//         b.png
+//
+//	data/
+//	  foo.txt
+//	  img/
+//	    a.png
+//	    b.png
+//
```

Comment:
Seems like intelij keeps making this change auto-magically.

Reply:
Can you ask inteliJ to ignore the bindata file, if we regenerate the asset again, there might be a battle between them :-<

Reply:
Yeah good point. Not sure what happened that it started to trigger this though..

I've added this file to the no-code-reformat file list in intelij. Seems to have done the job !